### PR TITLE
antistorm.eu

### PR DESCRIPTION
**Pull request by FadeMind**
https://antistorm.eu/#radar
https://antistorm.eu/#storm
https://antistorm.eu/obecna-pogoda.html

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -863,6 +863,7 @@ INVERT
 #map-area
 #map-controls-area
 #controlPlayPauseIcon
+img[src*="data:image/png;base64,iVBORw"]
```
Comment (FadeMind) [inline src/config/dynamic-theme-fixes.config:866]: @Gusted

**Comment by FadeMind** [comment]
CI Checker cannot handle recent changes. Reverted. 
Self extension in Edge working well with it.

**Comment by Gusted** [comment]
> CI Checker cannot handle recent changes. Reverted.

That's definitely a bug. It's probably recognizing the `,` inside the `src=""`.

**Comment by Gusted** [comment]
You can include it for now - Il be fixing that bit of code later today.

**Comment by FadeMind** [comment]
Done. 
Regards

**Comment by Gusted** [comment]
Please rebase the branch onto the master branch.

**Comment by FadeMind** [comment]
@Gusted ready to go

**Comment by FadeMind** [comment]
added meteograms inverts

**Review by Gusted — APPROVED** [review]

**Pull request merged**